lib/devise/strategies/passwordless_authenticatable.rb

```diff
@@ -40,7 +40,7 @@ def authenticate!
             UserMailer.with(email: email.downcase, full_name: user.full_name, url:, token_expiry: token_expiry.localtime.to_fs(:govuk)).sign_in_email.deliver_later(queue: "priority_mailers")
             raise LoginIncompleteError
           else
-            # UserMailer.with(recipient: email.downcase).access_info_email.deliver_later(queue: "priority_mailers")
+            UserMailer.with(recipient: email.downcase).access_info_email.deliver_later(queue: "priority_mailers")
```

Comment:
would it be worth passing the user instance as recipient so the mailer can do `.associate_with(recipient)` but we pass only the user.email to the template?

Reply:
@leoapost fair enough!